macros/console/close-vs-erase.mdx

```diff
@@ -0,0 +1,9 @@
+---
+macro: close-vs-erase
+---
+
+|                                  | **Closing your account**                                                                                                                               | **Erasing your personal data (GDPR)**                                                                                         |
+|----------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| **Use when**                     | You want to temporarily stop using Scaleway  without erasing your account’s settings  (e.g., billing information, invoices, security settings, etc.).  | You want to erase your account and all related data permanently.  This action is irreversible.                              |
```

Comment:
```suggestion
| **Use when**                     | You want to temporarily stop using Scaleway  without erasing your account’s settings  (e.g., billing information, invoices, security settings, etc.).  | You want to erase your account and all related data permanently. This action is irreversible.                              |
```

Reply:
I don't know if spacing displays weirdly because the table displays weirdly here, but there are sometimes two spaces between words (if you could check if it's a table issue or indeed a spacing issue)